modules/spring-config/src/main/java/org/mule/runtime/config/spring/SpringRegistryLifecycleManager.java

```diff
@@ -129,7 +131,9 @@ public SpringContextDisposePhase()
                     OutboundRouter.class,
                     Transformer.class,
                     MuleContext.class,
-                    ServerNotificationManager.class});
+                    ServerNotificationManager.class,
```

Comment:
May be possible to remove when changing the approach to provide the Service to the MuleContext